category-committed.php

```diff
@@ -0,0 +1,313 @@
+          <h4 class="font-size-9 text-uppercase font-weight-bold mb-2 mb-s-0"><?php echo $meta['_cmb_standfirst'][0]; ?></h4>
+          <h3 class="font-size-13 font-weight-semibold js-fix-widows"><?php the_title(); ?></h3>
+        </div>
+        <div class="flex-grid-item flex-offset-s-0 flex-item-s-12 flex-item-xxl-6">
+          <?php the_post_thumbnail( 'col12-16to9', array( 'class' => 'index-post-thumbnail' ) ); ?>
+        </div>
+        <div class="flex-grid-item flex-offset-s-0 flex-item-s-12 flex-offset-l-2 flex-item-l-8 flex-offset-xl-3 flex-item-xl-7 flex-offset-xxl-3 flex-item-xxl-6 mt-4 mb-4 mobile-mt-4 mobile-mb-4">
+          <iframe width="100%" height="115" scrolling="no" frameborder="no" allow="autoplay" src="https://w.soundcloud.com/player/?url=<?php echo urlencode( $meta['_cmb_sc'][0] ); ?>&color=%23ffab70&inverse=true&auto_play=false&show_user=false&show_artwork=false"></iframe>
+        </div>
+        <div class="flex-grid-item flex-offset-s-0 flex-item-s-12 flex-offset-l-2 flex-item-l-8 flex-offset-xl-3 flex-item-xl-7 flex-offset-xxl-3 flex-item-xxl-6 font-serif mb-4 text-paragraph-breaks committed__serif-medium">
+          <?php the_content(); ?>
+        </div>
+      </article>
+    <?php
+  }
+}
+?>
+    </div>
+  </section>
+  <div>
+    <div class="committed-credits__background pb-6 pt-6 font-color-white">
+      <div class="flex-grid-row mb-6 mb-s-5 mt-m-0 mt-4 committed-credits__logo-row">
+        <img src="<?php echo get_bloginfo( 'stylesheet_directory' ) . '/dist/img/specials/committed/committed-logo-white.png'; ?>" alt="Portrait of Rivkah Brown" class="committed-credits__logo"/>
+      </div>
+      <div class="flex-grid-row committed-credits__info-row font-size-10">
+
+        <!-- Reporters -->
+        <div class="flex-grid-item flex-item-m-4 flex-item-xxl-3 flex-item-s-8 mb-s-5">
+          <div class="flex-grid-row font-weight-bold mb-1">Reporters</div>
+          <div class="committed-credits__reporters flex-grid-row">
+            <div class="mr-2">
+              <img src="<?php echo get_bloginfo( 'stylesheet_directory' ) . '/dist/img/specials/committed/committed-rivkah.jpg'; ?>" alt="Portrait of Rivkah Brown" class="committed-credits__logo"/>
+              <p>Rivkah Brown</p>
+            </div>
+            <div class="">
+              <img src="<?php echo get_bloginfo( 'stylesheet_directory' ) . '/dist/img/specials/committed/committed-clare-logo.jpg'; ?>" alt="Portrait of Clare Hymer" class="committed-credits__logo"/>
+              <p>Clare Hymer</p>
+            </div>
+          </div>
+        </div>
+
+        <!-- Producers -->
+        <div class="flex-grid-item flex-item-m-4 flex-item-l-3 flex-item-xxl-3 flex-item-s-8 mb-s-5">
+          <div class="font-weight-bold mb-1">Producers</div>
+            <p class="mb-2"><?php echo nl2br( htmlspecialchars( $credits['producers'] ) ); ?></p>
```

Comment:
Not sure you need the `nl2br` & `htmlspecialchars` combo here. If you do need formatting likely using the apply_filters the_content filter would be best. That is a core Wordpress way to do something similar. If you search for that in the codebase you should find some examples

Reply:
this pattern made sense to me in the moment, but it's not user entered data so your method makes way more sense